CMakeLists.txt

```diff
@@ -15,9 +15,13 @@ list(APPEND CMAKE_MODULE_PATH ${PROJECT_SOURCE_DIR}/cmake)
 include(psi4OptionsTools)
 option_with_default(MAX_AM "The maximum gaussian angular momentum to compile" 8)
 option_with_default(CMAKE_BUILD_TYPE "Build type (Release or Debug)" Release)
-option_with_flags(ENABLE_XHOST "Enables processor-specific optimization" ON
-                  "-xHost" "-march=native")
-
+if(CMAKE_CXX_COMPILER_ID MATCHES Intel)
+option_with_flags(ENABLE_XHOST "Enables processor-specific optimization (with MSVC, it enables AVX2 instructions)" ON
+                  "-xHost" "-march=native" "/arch:AVX2")
+else()
+option_with_flags(ENABLE_XHOST "Enables processor-specific optimization (with MSVC, it enables AVX2 instructions)" ON
+                  "-march=native" "-xHost" "/arch:AVX2")
```

Comment:
Order matters?

Reply:
sadly, each compiler recognizes the other's xhost-like flag, though with different meanings.

Reply:
net result is a language error! or no xhost activated. no idea why it's popping up lately, but Susi found it on psi4, so we fixed it there and libxc.